File: Backend/HoneyShop.Model/Enums/ProductStatusEnum.cs
using System.ComponentModel;

namespace HoneyShop.Model.Enums;

public enum ProductStatusEnum
{
    [Description("В наличии")]
    InStock = 1,
    [Description("Нет в наличии")]
    OutOfStock = 2,
    [Description("Нет данных")]
    NoData = 3
}

File: Client/HoneyClient/src/app/data/enum/ProductStatusEnum.ts
export enum ProductStatusEnum {
  InStock = 1, // В наличии
  OutOfStock = 2, // Нет в наличии
  NoData = 3 // Нет данных
}

export const ProductStatusEnumLabels = {
  [ProductStatusEnum.InStock]: 'В наличии',
  [ProductStatusEnum.OutOfStock]: 'Нет в наличии',
  [ProductStatusEnum.NoData]: 'Нет данных'
};
